sdk-java/src/main/java/ly/count/sdk/java/internal/CoreFeature.java

```diff
@@ -4,8 +4,8 @@
 import java.util.Map;
 
 public enum CoreFeature {
-    Sessions(1 << 1, ModuleSessions::new),
-    Events(1 << 2, ModuleEvents::new),
+    Events(1 << 1, ModuleEvents::new),
```

Comment:
This is also needed to change because module events should initialize before module sessions. Because module sessions are trying to read timed events and put them to module events. But it was not before. Now it is also fixed with that. Also request order for device id change has fixed with that